website/src/assets/site.css

```
@@ -1462,6 +1479,10 @@ section.hero::after {
   color: var(--subtle);
   will-change: transform, opacity;
 }
+.hero-public svg,
+.hero-public i {
+  color: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)